lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -99,6 +113,13 @@ def get(self, request, *args, **kwargs):
         show_handouts = is_enrolled or is_staff or allow_public
         handouts_html = get_course_info_section(request, request.user, course, 'handouts') if show_handouts else ''
 
+        welcome_message_html = None
+        if get_course_tag(request.user, course_key, PREFERENCE_KEY) != 'False':
+            if LATEST_UPDATE_FLAG.is_enabled(course_key):
```

Comment:
Also, doing a brief look into this, I could only find it listed as true for a handful of courses back in 2017. Marco was the one who created all those so it might be worthwhile to reach out to him and see if we might not need to support this anymore in the MFE.